hawkular-wildfly-agent/src/main/java/org/hawkular/agent/monitor/service/MonitorService.java

```diff
@@ -976,6 +958,31 @@ public String getTenantId() {
         return this.storageAdapter.getStorageAdapterConfiguration().getTenantId();
     }
```

Comment:
replace the content of this method with this:
        return this.configuration.getStorageAdapter().getTenantId()